scapy/autorun.py

```diff
@@ -79,7 +79,7 @@ def autorun_commands(cmds, my_globals=None, ignore_globals=None, verb=0):
             pass
     finally:
         conf.verb = sv
-    return _
+    return _  # noqa: F821
```

Comment:
I doubt this will do anything :/

Reply:
Changing this line to something else broke UTScapy. I did not manage to find a better fix ..